api/routers/feature.py

```diff
@@ -0,0 +1,205 @@
+from typing import Annotated
+
+import datastore_pb2 as dstore
+import formatters
+from fastapi import APIRouter
+from fastapi import Path
+from fastapi import Query
+from geojson_pydantic import Feature
+from geojson_pydantic import FeatureCollection
+from grpc_getter import get_obs_request
+from grpc_getter import get_spatial_extent
+from shapely import geometry
+from utilities import get_datetime_range
+
+router = APIRouter(prefix="/collections/observations")
+
+
+@router.get(
+    "/items", tags=["Collection items"], response_model=Feature | FeatureCollection, response_model_exclude_none=True
+)
+async def search_timeseries(
+    bbox: Annotated[str | None, Query(example="5.0,52.0,6.0,52.1")] = None,
```

Comment:
Let's make this mandatory, even though that probably contradicts the spec. I think this will lead to better behavior of our users...

Reply:
For the most part I do agree, but in the case where on is looking up on or more specific platform(s) it would be user friendly to not have to provide the bbox.
Does fastAPI have "OneOf" functionality we could use? Or would we have to create a model for the request?